src/main/java/carpet/logging/HUDLogger.java

```diff
@@ -8,21 +8,26 @@
 public class HUDLogger extends Logger
 {
     static Logger stardardHUDLogger(String logName, String def, String [] options)
+    {
+        return stardardHUDLogger(logName, def, options, false);
+    }
+
+    static Logger stardardHUDLogger(String logName, String def, String [] options, boolean strictOptions)
     {
         // should convert to factory method if more than 2 classes are here
         try
         {
-            return new HUDLogger(LoggerRegistry.class.getField("__"+logName), logName, def, options);
+            return new HUDLogger(LoggerRegistry.class.getField("__"+logName), logName, def, options, strictOptions);
         }
         catch (NoSuchFieldException e)
         {
             throw new RuntimeException("Failed to create logger "+logName);
         }
     }
 
-    public HUDLogger(Field field, String logName, String def, String[] options)
+    public HUDLogger(Field field, String logName, String def, String[] options, boolean strictOptions)
```

Comment:
I would include the constructor that doesn't have the strictOptions param and default it to false for potential API compatibility, and lets mark it as @Deprecated